src/main/resources/public/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
```

Comment:
was there a reason to opt for this solution over using vue-cli to do the project setup?

Reply:
since I understood the UI was very "dumb" (only a table with a detail pane and an edit), I tried to reduce the number of technologies it needs. The advantage of this (lean) approach is to delay decisions reducing compromises with X or Y.
That said, there's no need to minify (only 59 lines of JS), bundle, run babel, or any other typical plugin. No need to use Node at all.
Of course, this could be changed as soon as requirements come in, but the UI code wouldn't be thrown away.
If the app doesn't grow much on the UI side, then it was spared effort (although for the sake of exercise/learning I could have gone further).